cli.js

```
@@ -48,22 +48,22 @@ if (process.stdout.isTTY) {
 
 let timeout;
 
-api((err, result) => {
-	if (err) {
-		throw err;
-	}
-
-	data = result;
-
-	// exit after the speed has been the same for 3 sec
-	// needed as sometimes `isDone` doesn't work for some reason
-	clearTimeout(timeout);
-	timeout = setTimeout(() => {
-		data.isDone = true;
-		exit();
-	}, 5000);
+api()
+	.forEach(result => {
+		data = result;
+		// exit after the speed has been the same for 3 sec
+		// needed as sometimes `isDone` doesn't work for some reason
+		clearTimeout(timeout);
+		timeout = setTimeout(() => {
+			data.isDone = true;
+			exit();
+		}, 5000);
 
-	if (data.isDone) {
-		exit();
-	}
-});
+		if (data.isDone) {
+			exit();
+		}
+	})
+	.then(() => exit())
+	.catch(err => {
+		throw err;
```

This needs to be:

```js
console.error(err.message);
process.exit(1);
```

Right now, errors are silenced.